features/webcrypto.yml

```diff
@@ -1,7 +1,11 @@
-draft_date: 2024-08-19
-name: Web Cryptography API
-description: TODO
-spec: https://w3c.github.io/webcrypto/
+name: Web Cryptography
```

Comment:
Ah, I got really confused about the name and identifier. Since we're calling this Web Cryptography, let's set the identifier/filename to `web-cryptography.yml`. Then I think this will be ready to merge. Thank you!